dspace-api/src/main/resources/Messages.properties

```diff
@@ -120,3 +120,4 @@ org.dspace.app.rest.exception.RESTEmptyWorkflowGroupException.message = Refused
 org.dspace.app.rest.exception.EPersonNameNotProvidedException.message = The eperson.firstname and eperson.lastname values need to be filled in
 org.dspace.app.rest.exception.GroupNameNotProvidedException.message = Cannot create group, no group name is provided
 org.dspace.app.rest.exception.GroupHasPendingWorkflowTasksException.message = Cannot delete group, the associated workflow role still has pending tasks
+org.dspace.app.rest.exception.PasswordNotValidException.message = New password is invalid. Valid passwords must have at least 8 characters long!
```

Comment:
Could we slightly modify this text to say:
"New password is invalid. Valid passwords must be at least 8 characters long!"